workspace/apps/pidp/src/app/features/access/pages/bc-provider-application/bc-provider-application.page.html

```diff
@@ -58,23 +58,26 @@ <h3>
             <ol>
               <li>
                 Go to Microsoft account and select
-                <strong>Sign in</strong>.
+                <strong>Sign in</strong>. Please use your BCProvider login and
+                not your vendor login?
               </li>
               <li>
-                Type the email, phone number, or Skype sign-in that you use for
-                other services(Outlook, Excel, etc.), then select
+                Enter in your email address (example:
+                first.lastname&#64;bcprovider.ca) then select
                 <strong>Next</strong>.
               </li>
               <li>
                 Type your password and select the
                 <strong>Keep me signed in box</strong> if you want to go
-                straight into your account next time (not recommended for shared
-                computers).
+                straight into your account next time.
+              </li>
+              <li>
+                Select <strong>Sign in</strong>, You may need to authenticate
```

Comment:
Another syntactical error from the requirements
```suggestion
                Select <strong>Sign in</strong>. You may need to authenticate
```